.buildkite/pipelines/quality-gates/emergency/pipeline.tests-production.yaml

```diff
@@ -38,3 +25,14 @@ steps:
     agents:
       # How long can this agent live for in minutes - 25 hours
       instanceMaxAge: 1500
+
+  - label: ":eye: Alert based quality gates"
+    trigger: "serverless-quality-gates"
+    build:
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-production.yaml)"
+      env:
+          TARGET_ENV: production
+          SERVICE: kbn-alert-quality-gate 
```

Comment:
Can you help me understand this tag bit better?
The [docs](https://github.com/elastic/serverless-quality-gates?tab=readme-ov-file#open-alerts-for-a-service-check_alerts) say
> In essence, it searches for the presence of SERVICE in the alert tags

I don't know how alert tags are set, are alerts taking over the tags from their corresponding rule? Or is there a separate mechanism for alert tags?
However, when checking our production overview cluster I wasn't able to find alerts or rules with the `kbn-alert-quality-gate` tag. Is this supposed to already match something?